build/Makefile

```diff
@@ -94,6 +94,8 @@ ${PDO_ENCLAVE_CODE_SIGN_KEY} :
 keys : $(PYTHON_DIR)
 	. $(abspath $(DSTDIR)/bin/activate) ; \
 		for i in 1 2 3 4 5 ; do $(KEYGEN) --keyfile $(DSTDIR)/opt/pdo/keys/eservice$${i} --format skf; done
+	. $(abspath $(DSTDIR)/bin/activate) ; \
```

Comment:
comment:
build/makefile is missing a "pip install lmdb" at the beginning
which is necessary for the eservice/pdo/eservice/scripts/SServiceCLI.py script

Reply:
this will be solved in a separated PR